.env.development

```diff
@@ -1,4 +1,5 @@
+ADMIN_HOST='http://localhost:3000'
 API_SERVICE_BACKEND_URL_OPTIONS={"uk":"http://localhost:3000","xi":"http://localhost:3000" }
-SERVICE_DEFAULT=uk
 PORT=3003
-ADMIN_HOST='http://localhost:3000'
+SERVICE_DEFAULT=uk
```

Comment:
I've just noticed that this is just `SERVICE` in the backend ... nothing just noticed it :)

Reply:
:tada: